helm-chart-sources/pulsar/templates/function/function-statefulset.yaml

```diff
@@ -194,6 +194,12 @@ spec:
             port: {{ .Values.function.probe.port }}
           initialDelaySeconds: {{ .Values.function.probe.initial }}
           periodSeconds:  {{ .Values.function.probe.period }}
+        livenessProbe:
+          httpGet:
+            path: {{ .Values.function.probe.metricsPath | default "/metrics" }}
+            port: {{ .Values.function.probe.port }}
+          initialDelaySeconds: {{ .Values.function.probe.initial }}
+          timeoutSeconds: {{ .Values.function.probe.period }}
```

Comment:
I notice that for the Pulsar Admin Console, we have the following in the values file:

```yaml
  probe:
    liveness:
      initialDelaySeconds: 15
      timeoutSeconds: 15
    readiness:
      initialDelaySeconds: 5
      timeoutSeconds: 3
```
In the case of the function worker statefulset, do you think it makes sense to always have the same configuration for the liveness and readiness probes? I am not familiar enough with function workers to have a strong opinion here, but I wanted to ask before we merge this.

Reply:
Spoke with @zzzming offline. This is the correct implementation.